lasagne/init.py

```diff
@@ -4,13 +4,14 @@
 Examples
 --------
 >>> from lasagne.layers import DenseLayer
->>> from lasagne.init import Constant, Glorot
+>>> from lasagne.init import Constant
 >>> l1 = DenseLayer((100,20), num_units=50, W=GlorotUniform(), b=Constant(0.0))
```

Comment:
It should be `Constant, GlorotUniform`. It still works without the import statement because it's the current module, but for people to be able to copy/paste it, we should include it.